util/execute.go

```diff
@@ -25,3 +25,12 @@ func Execute(args ...string) {
 		log.Fatalf("%s", output)
 	}
 }
+
+// ExecuteWithOutput runs the given strings as a command and returns the output.
+func ExecuteWithOutput(args ...string) string {
```

Comment:
Would propose following the pattern of `exec.Command`: make the first string argument required. Makes this harder to misuse and the same level of easiness to use correctly.

Reply:
Good idea. Will do.